### ISSUE TITLE
QUestion: Running without Cuda

### ISSUE THREAD

**Issue by rosanajurdi** [chengdazhi/Deformable-Convolution-V2-PyTorch#84]
Hello, 

Is it possible to create a deformed convolutional layer that runs without Cuda ? Time computation is not an issue.

Kind Regards, 
Rosana

**Comment by giangbang** [comment]
Yes you can, have a look at this [repo](https://github.com/giangbang/Pure-Pytorch-implementation-Deformable-Convolution.git) if you still wish to run deformable on cpu.
